app/views/shared/_attributes.html.erb

```diff
@@ -1,8 +1,11 @@
-      <table class="table">
+<% include_title ||= false # title is part of the exterior template in common cases %>
```

Comment:
Strictly speaking, no.  But if we (or our consumers) want to reuse the template in a display that includes the title as just the first of the set of attributes, then this conditional saves them from having to fork and override the whole template.  In general we need to move towards more configurable displays, ideally with the fields to be displayed enumerated outside of the templates themselves (similar to the way blacklight itself does it).

It doesn't seem to be the type of coverage that coveralls cares about, but you are correct that nothing is testing whether this conditional is honored.  Willing to add a test.  Also willing to defer.